assets/js/components/ExecutionResults/ExecutionResultsPage.jsx

```diff
@@ -64,6 +66,7 @@ function ExecutionResultsPage() {
       catalog={catalog}
       catalogError={catalogError}
       executionLoading={executionLoading}
+      executionRunning={runningStates.includes(executionData?.status)}
```

Comment:
Minor thing, but maybe we could move `executionStarted` paremeter down below here, to group the similar things together